tcg_rim_extensions.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package swid
```

Comment:
Could you please help me clarify, the PR says, fixes the issue to remove PSA Endorsements while this adds the new TCG RIM Extensions ?

Should this not be in a separate PR to fix another open item?

Reply:
This PR is self-contained and addresses the issue reflected in its title ("Add TCG RIM extensions").  ext-tcg-rim branches off remove-psa, that's why you are seeing commits from PR 39 too.  They have already been merged, so they are effectively no-ops.

Reply:
@thomas-fossati One more question: Why TCG RIM extensions should not be part of a profile, but sitting here as first class citizens ?